Automattic/wp-calypso
# Extend the Woo GTM tracking in Calypso to also fire in the woo express stepper flow

As we don't get enough data in the Woo GTM container for trial upgrades (users going from a free woo express trial to a paid plan) yet, we should add the Woo GTM container and track users earlier in the flow. E.g. in the stepper flow here: https://wordpress.com/setup/wooexpress/

Related to # 1749-gh-Automattic/martech

## Proposed Changes

* Introduce a new method `recordGTMDatalayerEvent` and load the Woo GTM container in the same.
* Call the method in the stepper flow step changes routine

## Testing Instructions



* Install the GADebugger Chrome Extension and enable
* Switch to the branch on this PR and start Calypso
* Visit /setup/wooexpress/ and cycle through the various steps
* Open Chrome Console and notice that you can see Tag manager events for the various steps

## Pre-merge Checklist
